capi/CMakeLists.txt

```diff
@@ -9,18 +9,20 @@
 # See the COPYING file for more information.
 ################################################################################
 
-if(GEOS_BUILD_FROM_GIT)
-  add_custom_target(geos_revision_header
```

Comment:
Replacing the target w/ plain command changes the procedure a bit
AFAIR, previous version re-generated the info on every build, not every re-configuration.

Reply:
I think it should be regenerated on each build, or just left out. A potentially incorrect hash seems to have more confusion potential than no hash.

Reply:
Well, generating every build has it's own downside, which Sandro alluded to on channel, which is the regen can end up kicking off large scale rebuilds over the code when not really necessary. I'm inclined to remove the whole thing.

Reply:
> I'm inclined to remove the whole thing.

Agreed

Reply:
>  regen can end up kicking off large scale rebuilds over the code 

It shouldn't, if `geos_revision.h` promises to stick to `geos_ts.cpp` only.

> I'm inclined to remove the whole thing.

Sweet

Reply:
Harsher approach at #174